### DISCUSSION TITLE
V1.7.4 and deactivate I2S

### DISCUSSION THREAD

**Discussion by mircose** [pschatzmann/ESP32-A2DP#518] [Q&A] [answered]
Hi,
I need to deactivate A2DP I2s to use another library and then be able to reactivate it.
in relase 1.7.4 there is a compile option A2DP_I2S_SUPPORT to deactivate I2S
there is any examples to deactivate e reactivate I2S?

**Comment by pschatzmann** [comment, marked as answer]
The best way is to manage I2S outside of A2DP and do the output of the audio using callbacks. See [this example](https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-basic-api/basic-a2dp-i2s/basic-a2dp-i2s.ino)
When you register the callback you can deactivate the i2s output...
